app/models/github/organization.rb

```diff
@@ -0,0 +1,127 @@
+class Github::Organization
+  include Mandate
```

Comment:
You might also like to `include Singleton` here and use `.instance` instead of `.new` in other places.